README.md

```diff
@@ -12,16 +12,56 @@ results, taking inspiration from
 
 ## Example
 
-Here's a trivial example:
+Let's say you have the following Rust binary called `simple`:
+
```

Comment:
While I know some people start with the README, I've also been burned too many times by either stale READEMEs or going through hoops to single source the README and lib,rs.  I think I'd like to keep this to the original content

Reply:
Actually, I think I found a way to include simple.md. You need to provide a permalink with a line range and then it gets inserted.

Reply:
I'm not seeing that reflected at https://github.com/SUPERCILEX/trycmd/tree/docs

Reply:
Oh, though actually I just realized this doesn't render the file as markdown. Hmmm

Reply:
Eh, I'll revert

Reply:
Thanks!  That helps!

I agree that without rendering this as markdown, its defeating the purpose.  We could just include only the code sample and the output block

Reply:
> We could just include only the code sample and the output block
Actually yeah, I like that.